Parse/TPTP.cpp

```diff
@@ -4976,16 +4976,22 @@ void TPTP::vampire()
     else if (lr == "skip") {
       skip = true;
     }
+    else if (lr == "uncomputable") {
+      uncomputable = true;
+    }
     else {
       PARSE_ERROR("'left', 'right' or 'skip' expected",getTok(0));
     }
-    env.colorUsed = true;
+    if (!uncomputable) env.colorUsed = true;
     Signature::Symbol* sym = pred
                              ? env.signature->getPredicate(env.signature->addPredicate(symb,arity))
                              : env.signature->getFunction(env.signature->addFunction(symb,arity));
     if (skip) {
       sym->markSkip();
     }
+    else if (uncomputable) {
+      sym->markUncomputable();
+    }
     else {
       sym->addColor(color);
```

Comment:
Getting
``Parse/TPTP.cpp:4996:20: warning: ‘color’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 4996 |       sym->addColor(color);``
 here with `gcc (Ubuntu 9.3.0-10ubuntu2) 9.3.0`. Could we add some default value at line 4967?